studio/schemas/objects/jobPosting.ts

```diff
@@ -0,0 +1,53 @@
+import { defineType } from "sanity";
+
+import { isInternationalizedString } from "studio/lib/interfaces/global";
+import { firstTranslation } from "studio/utils/i18n";
+
+export const jobPostingID = "jobPosting";
+
+const jobPosting = defineType({
+  name: jobPostingID,
+  title: "Job Posting",
+  type: "object",
+  fields: [
+    {
+      title: "Role",
+      name: "role",
+      type: "internationalizedArrayString",
+      description: "The name of the role",
+    },
+    {
+      title: "Location",
+      name: "location",
+      type: "reference",
+      to: [{ type: "companyLocation" }],
+      description: "Where is the role located?",
+    },
+    {
+      title: "Recruitee ad URL",
+      name: "recruiteeAdUrl",
+      type: "url",
+      description: "URL to Recruitee ad",
```

Comment:
You should add a rule here:

```
description: "Url to the recrutee post. Please enter full please enter the full URL, including 'https://'. For example, 'https://www.example.com'.",suggestion
validation: (rule) =>
        rule.uri({
          scheme: ["http", "https"],
          allowRelative: false,
        }),
```
I believe we should make all fields required to ensure that job postings are complete and don't lack essential information before being shared

Reply:
Fixed in https://github.com/varianter/variant.no/pull/908/commits/c29246e8d078be00cf34f2504f4464f5920e404c